src/Moryx.Runtime.WinService/Moryx.Runtime.WinService.csproj

```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Provides runtime mode for running as Windows Service.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>windows service;command line parser;moryx;iiot;iot</PackageTags>
```

Comment:
command line parser should be not used as tag here

maybe: "MORYX;Runtime;Server;Windows Service"